tests/load/smoke.py

```diff
@@ -0,0 +1,54 @@
+from locust import HttpLocust, TaskSet, task
+
+
+class SmokeBehavior(TaskSet):
+    """
+    Request behavior that represents some standard production behavior.
+    Tasks uris are hard-coded for now, but could be stored or even dynamically
+    generated later.
+    See https://bugzil.la/1186085
+    """
+
+    @task(265)
```

Comment:
Ah, cool, thanks! Could you make the parameter passed as `@task(weight=265)` so it's easier to understand what the number means?